index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main class="container">
+      <div class="todo-app">
+        <h2>To-Do List<img src="images/icon.png" alt="" / class="image"></h2>
+        <div class="row">
+          <input type="text" name="" id="inp_bx" placeholder="Add your Task" />
+          <button onclick="addTask()" id="add_btn">Add</button>
+        </div>
+        <ul id="list-container">
+          <!-- <li class="checked">Task 1</li>
+          <li>Task 2</li>
+          <li>Task 3</li> -->
+        </ul>
+      </div>
+    </main>
+    <script src="index.js"></script>
```

Comment:
We should place this script tag inside head using defer keyword, you can read more about placement of this script tag and what is async and defer.